proxy_agent/src/common/helpers.rs

```diff
@@ -14,7 +14,16 @@ use super::windows;
 static CURRENT_SYS_INFO: Lazy<(u64, usize)> = Lazy::new(|| {
     #[cfg(windows)]
     {
-        let ram_in_mb = windows::get_memory_in_mb();
+        let ram_in_mb = match windows::get_memory_in_mb() {
+            Ok(ram) => ram,
+            Err(e) => {
+                Error::new(
```

Comment:
what's this error used here?

Reply:
we need figure out how to surface this error first.

Reply:
by surface do you mean logging it?

Reply:
write_error and look into exception for lazy func

Reply:
if we wrap the function in a lazy_static!, we can use match when this function is called to raise exception.